docs/README.md

```diff
@@ -0,0 +1,69 @@
+# clownface
+
+[![Build Status](https://travis-ci.org/rdf-ext/clownface.svg?branch=master)](https://travis-ci.org/rdf-ext/clownface)
+[![NPM Version](https://img.shields.io/npm/v/clownface.svg?style=flat)](https://npm.im/clownface)
+
+Clownface is a graph traversal library inspired by [Gremlin](http://tinkerpop.apache.org/)
+which allows to query any [RDF dataset](https://rdf.js.org/dataset-spec/) in a concise and readable way.
+
+##  Introduction
+
+Querying an RDF graph can be challenging. Apart from trivial queries, e.g. simply listing all the
+objects connected to a subject, it is usually necessary to write multiple loop and validation
+conditions to retrieve an answer. Clownface is there to help.
+
+## Getting started
+
+The package is installed from npm:
+
+```
+npm i -S clownface
+```
+
+To start using it, an instance of [RDF/JS `DatasetCore`](https://rdf.js.org/dataset-spec/#datasetcore-interface) must be
+provided to the exported factory method
+
+<run-kit>
+
+```js
```

Comment:
I love the idea of creating a graph for the first example. Much cleaner and independent of loaders and stuff!
Further I propose to remove the name space part `{ foaf }` as it mixes concepts and might confuse. Its good to have it in the deepdive though!

Reply:
Fair enough